.github/workflows/main.yml

```diff
@@ -515,7 +533,11 @@ jobs:
       - uses: actions/download-artifact@v4
         with:
           name: mavsdk_server_macos.framework
-          path: ./build/macos/src/mavsdk_server/src/mavsdk_server.framework
+          path: ./build/macos/src/mavsdk_server/src/mavsdk_server_x64.framework
+      - uses: actions/download-artifact@v4
+        with:
+          name: mavsdk_server_macos.framework
+          path: ./build/macos/src/mavsdk_server/src/mavsdk_server_arm64.framework
       - name: Package
         run: bash ./src/mavsdk_server/tools/package_mavsdk_server_framework.bash
```

Comment:
Note that this script will have to be updated if you want to include the new builds in the final framework :+1:

Reply:
Yes, do you know how? I couldn't really find the docs on how the structure should be.

Reply:
I think what you did (after the comment) makes sense, but I don't really understand the other changes. Was there a need to update the dependencies and toolchain for this?

Reply:
I'm working on it...